src/router_core/management_agent_private.h

```diff
@@ -20,6 +20,6 @@
  * under the License.
  */
 
-void management_agent_handler(void *context, qd_message_t *msg, int link_id);
+void qd_management_agent_handler(void *context, qd_message_t *msg, int link_id);
```

Comment:
I think a more descriptive name is needed here.  Something like qd_router_agent_on_message.